lib/provider.js

```diff
@@ -385,15 +385,11 @@ class StorageProvider {
         return;
       }
       const { Execute: execute } = foundAction.definition;
-      execute(
-        this,
-        ...actionArgs,
-        this.loggingEnabled
-          ? callback
-          : (...args) => {
-              callback(...args.map(LogStatus.strip));
-            }
-      );
+      const thenHandlers = [val => callback(null, val), callback];
+      if (this.loggingEnabled) {
```

Comment:
This seems to be reverted now, isn't it? (previously we would map args if this.loggingEnabled is falsy)